packages/inter-protocol/src/feeDistributor.js

```diff
@@ -289,10 +290,16 @@ export const makeFeeDistributor = (feeIssuer, terms) => {
      */
     setDestinations: async newDestinations => {
       destinations = newDestinations;
-      shareConfig = makeShareConfig(destinations, terms.keywordShares);
+      shareConfig = makeShareConfig(destinations, keywordShares);
       // Run once immediately for these destinations.
       await schedulePayments();
     },
+
+    /** @param {Record<Keyword, bigint>} newShares */
+    setKeywordShares: newShares => {
+      mustMatch(newShares, M.recordOf(M.string(), M.nat()));
+      keywordShares = newShares;
```

Comment:
The original `keywordShares` is visible in `terms`. Is there any visibility for the updated value? It could be on the public facet or pushed to chain storage.

Reply:
I can put it on the public facet without significant more work.

Rendering the terms out of date / unreliable does seem awkward, but more challenging to address. Gotta think it over a bit.

Reply:
I agree that that would be a good goal, but I think we can leave it to address til later.

Reply:
I added it to the public facet.